src/store/actions/community.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable linebreak-style */
+import axios from 'axios'
+import { FETCH_COMMUNITY_DATA_REQUEST, FETCH_COMMUNITY_DATA_SUCCESS, FETCH_COMMUNITY_DATA_FAILURE } from './actionTypes';
+
+
+export const fetchCommunityDataRequest = () => ({ type: FETCH_COMMUNITY_DATA_REQUEST})
+export const fetchCommunityDataSuccess = (data) => ({ type: FETCH_COMMUNITY_DATA_SUCCESS, payload: data})
+export const fetchCommunityDataFailure = (error) => ({ type: FETCH_COMMUNITY_DATA_FAILURE, payload: error})
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required before '}'.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)